torchrl/envs/gym_like.py

```diff
@@ -133,18 +133,40 @@ def read_action(self, action):
         """
         return self.action_spec.to_numpy(action, safe=False)
 
-    def read_done(self, done):
+    def read_done(
+        self,
+        gym_done: bool,
+        termination: bool | None = None,
+        truncation: bool | None = None,
+    ) -> Tuple[bool, bool, bool]:
         """Done state reader.
 
-        Reads a done state and returns a tuple containing:
-        - a done state to be set in the environment
-        - a boolean value indicating whether the frame_skip loop should be broken
+        In torchrl, a `"done"` signal means that a trajectory is terminated
+        (what is referred to as `termination` in gymnasium).
+        Truncated means the trajectory has been interrupted.
 
         Args:
-            done (np.ndarray, boolean or other format): done state obtained from the environment
+            gym_done (np.ndarray, boolean or other format): done state obtained from the environment
+            termination (bool or None): termination signal. Defaults to `None`.
+            truncation (bool or None): truncation signal. Defaults to `None`.
+
+        Returns: a tuple with 3 boolean values,
+            - a done state to be set in the environment.
+            - a truncated state, possibly None if no truncation is provided.
+            - a boolean value indicating whether the frame_skip loop should be broken.
 
         """
-        return done, done.any() if not isinstance(done, bool) else done
+        if termination is not None:
+            gym_done = termination
```

Comment:
This would be a torchrl `done` no?

Reply:
I mean it's still correct, but just not to overload the variable if not returned directly (like in isaacgym for example)

Reply:
No you're right i did a replace all within the function
Thanks for the suggestions